server/webhook/issue.go

```diff
@@ -26,7 +26,7 @@ func (w *webhook) handleDMIssue(event *gitlab.IssueEvent) ([]*HandleWebhook, err
 
 	switch event.ObjectAttributes.Action {
 	case actionOpen:
-		if len(event.Assignees) > 0 {
+		if event.Assignees != nil && len(event.Assignees) > 0 {
```

Comment:
Go automatically checks if this is `nil` before checking its length, so this change is unnecessary

Reply:
Also, the changes in this file seem irrelevant to this PR. Please raise these changes in another issue/PR

Reply:
fixed here 4146d4ba7739302a496d23d0017c2e0f4ca30732